### ISSUE TITLE
look at jspm for import of npm modules

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#1835]
see https://jspm.io/
can we somehow hook into this to allow users to import modules from npm?

**Comment by jacobq** [comment]
Ah, I think I figured it out...should be using a `fetch` chunk:

```
%% fetch
js: https://cdn.plot.ly/plotly-latest.min.js
```

**Comment by jacobq** [comment]
> Do you need more help, or does the `fetch` chunk solve your problem?

It solves that problem. However, I guess I was hoping for an easier way to add UI elements, especially input controls. Looks like right now there is no built-in support for this; it has to be added just like it would be for any other web page, is that right?

**Comment by bcolloran** [comment]
yes for the time being that's correct. streamlining the story around building UIs is definitely something we're thinking about, but it's not there yet.
